src/orange/Constants.hh

```diff
@@ -4,6 +4,7 @@
 // SPDX-License-Identifier: (Apache-2.0 OR MIT)
 //---------------------------------------------------------------------------//
 //! \file orange/Constants.hh
+//! \todo move to corecel/
```

Comment:
+1

Reply:
Maybe we need another issue like https://github.com/celeritas-project/celeritas/issues/491 — we should reorganize before 1.0 as well as renaming from #721